src/SonarScanner.MSBuild/Program.cs

```diff
@@ -58,14 +58,19 @@ public static async Task<int> Execute(string[] args, ILogger logger)
                 logger.LogInfo("Usage: ");
                 logger.LogInfo(string.Empty);
                 logger.LogInfo(
-                    "  {0} [begin|end] /key:project_key [/name:project_name] [/version:project_version] [/d:sonar.key=value] [/s:settings_file]",
-                    System.AppDomain.CurrentDomain.FriendlyName);
+                    @"  {0} [begin|end] /key:project_key [/name:project_name] [/version:project_version] [/s:settings_file] [/d:sonar.login=token] [/d:sonar.{{property_name}}=value]",
+                    AppDomain.CurrentDomain.FriendlyName);
                 logger.LogInfo(string.Empty);
-                logger.LogInfo("    When executing the begin phase, at least the project key must be defined.");
-                logger.LogInfo("    Other properties can dynamically be defined with '/d:'. For example, '/d:sonar.verbose=true'.");
-                logger.LogInfo("    A settings file can be used to define properties. If no settings file path is given, the file SonarQube.Analysis.xml in the installation directory will be used.");
-                logger.LogInfo("    Only the token should be passed during the end phase, if it was used during the begin phase.");
-
+                logger.LogInfo("  - When executing the begin phase, at least the project key must be defined.");
+                logger.LogInfo("  - When a token is needed, it should be provided through 'sonar.login' parameter in both 'BEGIN' and 'END' steps. It should be the only provided parameter during the `END` step");
```

Comment:
I think the token needs to be given starting since last LTS https://docs.sonarqube.org/latest/setup/lts-to-lts-upgrade-notes/ , so "When a token is needed" can be replaced by "The authentication token should be provided..."